app/scripts/lib/id-management.js

```diff
@@ -25,13 +24,9 @@ function IdManagement (opts) {
   }
 
   this.signTx = function (txParams) {
-    //  calculate gas with custom gas multiplier
-    var gasMultiplier = this.configManager.getGasMultiplier() || 1
-    var gasPrice = new BN(ethUtil.stripHexPrefix(txParams.gasPrice), 16)
-    gasPrice = gasPrice.mul(new BN(gasMultiplier * 100, 10)).div(new BN(100, 10))
-    txParams.gasPrice = ethUtil.intToHex(gasPrice.toNumber())
-    // normalize values
 
+    // normalize values
+    txParams.gasPrice = ethUtil.intToHex(txParams.gasPrice)
```

Comment:
This is a legacy file that I only updated to remove the gasMultiplier from.  We don't actually sign txs from it anymore.